package copy.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Any reason to keep a hard copy of old package.json in git?

Reply:
No, added/pushed by mistake
I removed it